### PR TITLE
Changes to support ARM64 architecture

### PR DESCRIPTION
**Pull request by bpramod**
-> Changes to create a new binary for ARM64 architecture
-> CHanges to package newly created ARM64 binary folder

### CODE REVIEW

**Review by mearvind — APPROVED** [review]

File: VMBackup/main/fsfreezer.py [hunk 1]
```diff
@@ -150,7 +171,7 @@ def freeze_safe(self,timeout):
             self.logger.log(errMsg,True,'Warning')
         try:
             freeze_result = FreezeResult()
-            freezebin=os.path.join(os.getcwd(),os.path.dirname(__file__),"safefreeze/bin/safefreeze")
+            freezebin=os.path.join(os.getcwd(),os.path.dirname(__file__),self.safeFreezeFolderPath)
```
Comment (vityagi) [inline VMBackup/main/fsfreezer.py:174]: Can we also test the error we get when we run wrong architecture executable. And based on that can we fallback to the other one ?

File: VMBackup/setup.py [hunk 2]
```diff
@@ -188,6 +202,7 @@ def copy(src, dst):
 copybinary(workloadutils_folder, final_workloadutils_path)
 final_plugin_conf_path = final_folder_path + '/main'
 copybinary(binary_entry, final_binary_path)
+copybinary(binary_entry, final_Arm64binary_path)
```
Comment (vityagi) [inline VMBackup/setup.py:205]: is the same binary being copied to both folders ?

**Review by vityagi — COMMENTED** [review]
Some comments added

File: VMBackup/setup.py (hunk 2 above)
Comment (bpramod) [reply VMBackup/setup.py:205]: good catch....missed updating source path

File: VMBackup/main/fsfreezer.py (hunk 1 above)
Comment (bpramod) [reply VMBackup/main/fsfreezer.py:174]: actually, when we create the child process, we don't check whether that succeed....we were waiting for it to complete but it is getting crashed. it is tough to handle in such a way

**Review by vityagi — APPROVED** [review]
looks fine

**Pull request merged**